bundles/org.openhab.binding.ventaair/src/main/java/org/openhab/binding/ventaair/internal/message/action/AllActions.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ventaair.internal.message.action;
+
+/**
+ * Actions send by the device, containing information about the current device settings
+ *
+ * @author Stefan Triller - Initial contribution
+ *
+ */
+public class AllActions implements Action {
+    private boolean Power = true;
+    private int FanSpeed;
+    private int TargetHum = 65;
+    private int Timer;
+    private boolean Boost;
+    private boolean SleepMode;
+    private boolean ChildLock;
+    private boolean Automatic;
+    private int SysLanguage; // 3?
+    private int CleanLanguage; // 0?
+    private int TempUnit; // 0=Celsius, 1=Fahrenheit?
+    private int DisplayLeft;
+    private int DisplayRight;
+    private int Reset;
+    private int ConINet;
+    private boolean DelUser; // default false
```

Comment:
All variables should use lowerCamelCase and not start with a capitial letter.

Reply:
Yes, that is because they are needed for GSON. of course I could add an annotation to map all these names, but this would IMHO just clutter the code.